src/renderers/native/ReactNativeFiber.js

```diff
@@ -416,4 +417,11 @@ const ReactNative = {
 
 };
 
+if (typeof injectInternals === 'function') {
```

Comment:
Either one. Since this only gets called once, it should be fine to just check `typeof __REACT_DEVTOOLS_GLOBAL_HOOK__ !== 'undefined' && __REACT_DEVTOOLS_GLOBAL_HOOK__.supportsFiber` at inside of it.